app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -0,0 +1,271 @@
+                }
+              }
+            } else {
+              items(routes.value.size) { index: Int ->
+                HikingRouteItem(
+                    title = routes.value[index],
+                    altitudeDifference = 1000,
+                    isSuitable = index % 2 == 0,
+                    onClick = {
+                      // TODO : Navigate to the details of the selected hiking route
+                      Toast.makeText(
+                              context, "Hike details not implemented yet", Toast.LENGTH_SHORT)
+                          .show()
+                    })
+              }
+            }
+          }
+        }
+      },
+      sheetPeekHeight = 400.dp) {}
+}
+
+@Composable
+fun HikingRouteItem(
+    title: String,
+    altitudeDifference: Int,
+    isSuitable: Boolean,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+  Row(
+      modifier =
+          modifier
+              .fillMaxWidth()
+              .clickable(onClick = onClick)
+              .padding(16.dp, 8.dp)
+              .background(MaterialTheme.colorScheme.surface, RoundedCornerShape(8.dp))
+              .testTag(MapScreen.TEST_TAG_HIKE_ITEM),
+      verticalAlignment = Alignment.CenterVertically) {
+        Column(modifier = Modifier.weight(1f)) {
+          Text(
+              text = title,
+              style = MaterialTheme.typography.titleLarge,
+              fontWeight = FontWeight.Bold)
+          Spacer(modifier = Modifier.height(8.dp))
+
+          Row(
+              verticalAlignment = Alignment.CenterVertically,
+              modifier = Modifier.fillMaxWidth().height(IntrinsicSize.Min)) {
+                Box(
+                    modifier =
+                        Modifier.weight(1f)
+                            .fillMaxHeight()
+                            .background(
+                                MaterialTheme.colorScheme.primary.copy(alpha = 0.1f),
+                                RoundedCornerShape(4.dp)))
+
+                Spacer(modifier = Modifier.width(8.dp))
+
+                Column {
+                  Text(text = "Altitude difference", style = MaterialTheme.typography.bodySmall)
+                  Text(
+                      text = "${altitudeDifference}m",
+                      style = MaterialTheme.typography.bodyLarge,
+                      fontWeight = FontWeight.Bold)
+                }
+              }
+
+          Spacer(modifier = Modifier.height(4.dp))
+
+          Row(verticalAlignment = Alignment.CenterVertically) {
+            Icon(
+                imageVector = if (isSuitable) Icons.Default.Check else Icons.Default.Warning,
+                // The icon is only decorative, the following message is enough for accessibility
+                contentDescription = null,
+                // TODO : Replace suitable and challenging icon colors with theme colors
+                tint = if (isSuitable) Color(0xFF4CAF50) else Color(0xFFFFC107),
```

Comment:
Nitpick: If you do decide to remove todos and replace them in with issues, afaik there is already a task in the product backlog on adding a theme file, you could just add to this task to replace the colors in this section here.

Reply:
Done 👌🏻